recipes/lapack95/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "lapack95" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.netlib.org/lapack95/lapack95.tgz
```

Comment:
Do they provide a versioned URL or do we know of an FTP server that does? That would make tracking and updating to new releases easier

Reply:
That is the official website for it... unfortunately other sources that claim mirroring of it modified the code a lot to the point I am not sure if it would produce the same result.

Reply:
Ah ok. How do we know if a new version is released?

Reply:
Probably from periodic visits to their website. Latest release which is this one is from `Date:  November 30, 2000`.